zarnitsa/DataAugmenterInternally.py

```diff
@@ -55,13 +63,34 @@ def augment_column(
         to_aug = augment_column_method[aug_type](col, **kwargs)
         return to_aug if return_only_aug else pd.concat([not_to_aug, to_aug])
```

Comment:
`return_only_aug` is undefined

Reply:
`not_to_aug` is undefined too

Reply:
`return_only_aug` is undefined in `augment_dataframe` too

Reply:
my bad